app/models/concerns/search.rb

```diff
@@ -0,0 +1,10 @@
+module Search
+  extend ActiveSupport::Concern
+  module ClassMethods
+    def search(query)
+      lambda do |args|
+        where(query, "%#{args}%", "%#{args}%", "%#{args}%")
+      end
```

Comment:
Why do you need this in concern at all? This looks totally unnecessary. Just use ```where(query, "%#{args}%", "%#{args}%", "%#{args}%")``` where it is needed.